### ISSUE TITLE
"auth0 logs tail" date field is useless

### ISSUE THREAD

**Issue by joshw** [auth0/auth0-cli#418]
### Description

Each line in the `auth0 logs tail` output includes a `Date` column formatted as as a relative time.

"2 seconds ago" is computed relative to the log message vs the time that the message is displayed.  This makes sense for log queries, but in tail mode, it produces completely useless values (always a few seconds ago)   It needs to show absolute time instead.

### Reproduction

```
auth0 logs tail
```

produces lines such as
```
Success Exchange           Client Credentials for Access Token                       2 seconds ago        N/A                     ClientName
```

It should show something along the lines of this instead:
```
Success Exchange           Client Credentials for Access Token                       2022-05-03T21:47:08.899Z        N/A                     ClientName
```

### Environment

> Please provide the following:

- **Version of this library used:**

```
% auth0 --version
auth0 version 0.11.7 f9bcaf1b0d5c6c380022842b9707dfb8620b4890
```

- **Version of the platform or framework used, if applicable:**

n/a

- **Other relevant versions (language, server software, OS, browser):**

```
Darwin  20.6.0 Darwin Kernel Version 20.6.0: Wed Jan 12 22:22:42 PST 2022; root:xnu-7195.141.19~2/RELEASE_X86_64 x86_64
```

- **Other modules/plugins/libraries that might be involved:**

n/a

**Comment by willvedd** [comment]
**Update:** We've included fix for this in the [v1.0.0-beta.1](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0-beta.1) release. For those who are so inclined, we'd be happy to receive feedback on the beta before it goes GA, including feedback on this particular issue. Otherwise, we plan to close this issue out once v1.0.0 reaches GA. Again, appreciate everyone's patience!

**Comment by willvedd** [comment]
Fixed and released in our [v1.0.0](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0) release. We encourage folks to upgrade and give it a shot!
